sml/include/sml_log.h

```diff
@@ -35,11 +35,11 @@ extern "C" {
  */
 enum sml_log_level {
     SML_LOG_LEVEL_DEBUG = 1 << 0,     /**< Show debug messages. The debug messages will not be logged with SML is compiled in Release mode. */
-    SML_LOG_LEVEL_INFO = 1 << 1,     /**< Show info messages. */
-    SML_LOG_LEVEL_WARNING = 1 << 2, /**< Show warning messages. */
-    SML_LOG_LEVEL_ERROR = 1 << 3, /**< Show error messages. */
-    SML_LOG_LEVEL_CRITICAL = 1 << 4, /**< Show critical messages. */
-    SML_LOG_LEVEL_ALL = (SML_LOG_LEVEL_DEBUG | SML_LOG_LEVEL_INFO | SML_LOG_LEVEL_WARNING | SML_LOG_LEVEL_ERROR | SML_LOG_LEVEL_CRITICAL) /**< Show all messages. */
+        SML_LOG_LEVEL_INFO = 1 << 1,     /**< Show info messages. */
+        SML_LOG_LEVEL_WARNING = 1 << 2, /**< Show warning messages. */
+        SML_LOG_LEVEL_ERROR = 1 << 3, /**< Show error messages. */
+        SML_LOG_LEVEL_CRITICAL = 1 << 4, /**< Show critical messages. */
+        SML_LOG_LEVEL_ALL = (SML_LOG_LEVEL_DEBUG | SML_LOG_LEVEL_INFO | SML_LOG_LEVEL_WARNING | SML_LOG_LEVEL_ERROR | SML_LOG_LEVEL_CRITICAL) /**< Show all messages. */
 };     /**< The types of log levels */
```

Comment:
This report is an issue with older versions of uncrustify. If you use uncrustify with version >= 0.62, this won't be reported. In Soletta we have some enums with the style you are proposing, but we preferring to use the style proposed by a newer uncrustify.